app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -0,0 +1,28 @@
+# Thank you for completing your teacher training application
+
+You have submitted an application to:
+
+You can track the progress of your application here:
+https://apply-for-teacher-training.service.gov.uk/applications
```

Comment:
Can we add the real URL here?

Reply:
done, I used `candidate_interface_application_form_url`  (/candidate/application)